src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+        Test.startTest();
+        update addrs;
+        Test.stopTest();
+    
+        // verify that the Account and Contacts don't share the same address and it's new for the Account!
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c 
+        from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            system.assertEquals(null, acc.BillingStreet); //address information has been cleared because address is no longer the default
+            system.assertEquals(null, acc.BillingCity); //address information has been cleared because address is no longer the default
+            system.assertEquals(false, acc.is_Address_Override__c);
+            system.assertEquals(null, acc.Current_Address__c); //account no longer has a current address, since address is no longer the defaults
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Set account address override.  
+    verify: account's address matches override
+    **********************************************************************************************************/            
+    @isTest
+    public static void setAddrOverride() {
+        configSettings();
+        
+        // this creates a default Address for the Account
+        UTIL_UnitTestData_TEST.AccsWithAddrs accsAddrs = UTIL_UnitTestData_TEST.createTestAccsAddrs(2, UTIL_Describe.getAdminAccRecTypeID());
+        List<Address__c> addrs = accsAddrs.addrs;
+        
+        // create new non-default address
+        Address__c addrNew = new Address__c();
+        addrNew.MailingStreet__c = 'street override';
+        addrNew.MailingCity__c = 'city override';
+        addrNew.Default_Address__c = false;
+        // assign it to the second account
+        addrNew.Parent_Account__c = accsAddrs.accs[1].Id;
+        
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        insert addrNew;
+        
+        // set account's address override
+        accsAddrs.accs[1].Current_Address__c = addrNew.Id;
+        accsAddrs.accs[1].is_Address_Override__c = true;
+        
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        update accsAddrs.accs[0];
+    
+        // now let's update the default Address for both accounts
+        addrs[0].MailingStreet__c = 'New Street';
+        addrs[0].MailingCity__c = 'New City';
+        addrs[1].MailingStreet__c = 'New Street';
+        addrs[1].MailingCity__c = 'New City';
+
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        update addrs;
+        Test.stopTest();
+        
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.